arch/arm/boot/dts/xilinx/ni-bluefin.dts

```diff
@@ -75,6 +72,55 @@
 		full-duplex;
 		reg = <0>;
 	};
+
+	mdio0: mdio {
+		status = "okay";
+	};
+};
+
+&mdio0 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	switch: switch@0 {
+		compatible = "marvell,mv88e6085";
+		reg = <0x0>;
+		status = "okay";
+
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			port@0 {
+				reg = <0>;
+				label = "cpu";
+				phy-mode = "rgmii-id";
```

Comment:
Fixed an issue with phy-mode specification. Now using "rgmii-id" (rgmii with internal delays) as that appears to be the correct configuration for switch on Bluefin.